JetNews/app/build.gradle.kts

```diff
@@ -97,17 +97,30 @@ dependencies {
     implementation(libs.kotlin.stdlib)
     implementation(libs.kotlinx.coroutines.android)
 
-    implementation(libs.androidx.compose.foundation.layout)
-    implementation(libs.androidx.compose.material)
-    implementation(libs.androidx.compose.material.iconsExtended)
-    implementation(libs.androidx.compose.materialWindow)
-    implementation(libs.androidx.compose.animation)
-    implementation(libs.androidx.compose.ui.tooling.preview)
-    debugImplementation(libs.androidx.compose.ui.tooling)
-    implementation(libs.androidx.compose.runtime.livedata)
-
-    implementation(libs.accompanist.swiperefresh)
-    implementation(libs.accompanist.systemuicontroller)
+    // TODO: Replace when all samples use the same Material3 & Compose version
+    implementation("androidx.compose.animation:animation:1.3.0-beta01")
+    implementation("androidx.compose.foundation:foundation-layout:1.3.0-beta01")
+    implementation("androidx.compose.material:material-icons-extended:1.3.0-beta01")
+    implementation("androidx.compose.material3:material3:1.0.0-beta01")
+    implementation("androidx.compose.material3:material3-window-size-class:1.0.0-beta01")
+    implementation("androidx.compose.runtime:runtime-livedata:1.3.0-beta01")
+    implementation("androidx.compose.ui:ui-tooling-preview:1.3.0-beta01")
+    debugImplementation("androidx.compose.ui:ui-test-manifest:1.3.0-beta01")
+    debugImplementation("androidx.compose.ui:ui-tooling:1.3.0-beta01")
+//    implementation(libs.androidx.compose.animation)
+//    implementation(libs.androidx.compose.foundation.layout)
+//    implementation(libs.androidx.compose.material.iconsExtended)
+//    implementation(libs.androidx.compose.material3)
+//    implementation(libs.androidx.compose.materialWindow)
+//    implementation(libs.androidx.compose.runtime.livedata)
+//    implementation(libs.androidx.compose.ui.tooling.preview)
+//    debugImplementation(libs.androidx.compose.ui.test.manifest)
+//    debugImplementation(libs.androidx.compose.ui.tooling)
```

Comment:
Remove? Do you also want to add the dependencies to the .toml file or is that what the TODO is for?

Reply:
The todo is to remove the explicit dependencies and use these commented out versions again, once all Compose & Material3 dependencies align again